src/phpDocumentor/Descriptor/TocDescriptor.php

```diff
@@ -5,33 +5,25 @@
 namespace phpDocumentor\Descriptor;
 
 use phpDocumentor\Descriptor\TableOfContents\Entry;
+use phpDocumentor\Descriptor\Traits\HasDescription;
+use phpDocumentor\Descriptor\Traits\HasName;
 
 use function array_filter;
 
 final class TocDescriptor implements Descriptor
 {
-    /** @var string */
-    private $name;
+    use HasName;
+    use HasDescription;
```

Comment:
Description can be removed?

Reply:
The HasDescription trait is because the TocDescriptor uses the Descriptor interface, which requires getDescription. In a subsequent PR we should consider whether getDescription should be in the Descriptor interface